### PR TITLE
WT-10302 Convert tabs to spaces in dist/ files

### PR DESCRIPTION
**Pull request by bzm-jas**
Updated indentations of all files in dist/ scripts from tabs to spaces of 4.

### CODE REVIEW

**Comment by etienneptl** [comment]
Is it possible to have something (a script?) as part of `s_all` to check for those tabs and replace them? This way we can avoid the same work in the future!

**Comment by jiechenbo** [comment]
I agree @etienneptl , I think we should create a ticket to create the script, you can leave that to me 👍

File: dist/filelist [hunk 1]
```diff
@@ -47,16 +47,16 @@ src/btree/col_srch.c
...
 src/btree/row_modify.c
 src/btree/row_srch.c
-src/checksum/arm64/crc32-arm64.c	ARM64_HOST
-src/checksum/loongarch64/crc32-loongarch64.c	LOONGARCH64_HOST
-src/checksum/power8/crc32_wrapper.c	POWERPC_HOST
-src/checksum/power8/vec_crc32.c		POWERPC_HOST
-src/checksum/riscv64/crc32-riscv64.c	RISCV64_HOST
+src/checksum/arm64/crc32-arm64.c    ARM64_HOST
+src/checksum/loongarch64/crc32-loongarch64.c    LOONGARCH64_HOST
+src/checksum/power8/crc32_wrapper.c    POWERPC_HOST
```
Comment (jiechenbo) [inline dist/filelist:52]: It seems that we previously aligned the 2nd column and converting it to spaces has unaligned it, can we adjust with spaces?

File: dist/filelist [hunk 2]
```diff
@@ -134,41 +134,41 @@ src/os_common/os_fstream.c
...
-src/os_win/os_priv.c			WINDOWS_HOST
-src/os_win/os_setvbuf.c			WINDOWS_HOST
-src/os_win/os_sleep.c			WINDOWS_HOST
-src/os_win/os_snprintf.c		WINDOWS_HOST
-src/os_win/os_thread.c			WINDOWS_HOST
-src/os_win/os_time.c			WINDOWS_HOST
-src/os_win/os_utf8.c			WINDOWS_HOST
-src/os_win/os_winerr.c			WINDOWS_HOST
-src/os_win/os_yield.c			WINDOWS_HOST
+src/os_posix/os_dir.c            POSIX_HOST
```
Comment (jiechenbo) [inline dist/filelist:137]: same as before (ditto)

File: dist/s_funcs [hunk 3]
```diff
@@ -5,7 +5,7 @@ t=__wt.$$
 trap 'rm -f $t' 0 1 2 3 13 15
 
 # List of files to search.
-l=`sed -e '/^[a-z]/!d' -e 's/[	 ].*$//' -e 's,^,../,' filelist`
+l=`sed -e '/^[a-z]/!d' -e 's/[     ].*$//' -e 's,^,../,' filelist`
```
Comment (jiechenbo) [inline dist/s_funcs:8]: We will probably need to revise this regex, maybe we only need one space instead here

File: dist/s_python [hunk 4]
```diff
@@ -8,24 +8,24 @@ cd ..
 
 # Check Python coding standards: check for tab characters.
 # Ignore generated files.
-egrep '	' `find . -name '*.py'` |
+egrep '    ' `find . -name '*.py'` |
```
Comment (jiechenbo) [inline dist/s_python:11]: Looks like we want to intentionally have tabs here, so that we that we can check against tabs

**Review by jiechenbo — COMMENTED** [review]
There is a number of sed commands that previously used tab as a capture and has possibly changed wrongly to just 4 spaces. Happy to chat to revise it together 😄

**Review by jiechenbo — APPROVED** [review]
LGTM!

**Review by Siddhartha8899 — APPROVED** [review]

**Comment by Siddhartha8899** [comment]
Good Job! Make sure to resolve the merge conflicts. 🥇

**Pull request merged**